content/docs/studio/user-guide/account-and-billing/index.md

```diff
@@ -133,17 +102,56 @@ In this section, you can,
   GitHub app on additional organizations or repositories, or even remove the app
   from organizations or repositories where you no longer need it.
 
-## Delete account
+## Cloud connections
 
-If you delete your account, all the projects you own and the links that you have
-shared will be permanently deleted. So, click on `Delete my account` only if you
-are absolutely sure that you do not need those projects or links anymore.
+In this section, you can view, add and update credentials for cloud resources.
+These credentials are used to
+[fetch project data from data remotes](/doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials).
 
-<admon>
+To add new credentials, click `Add credentials` and select the cloud provider.
+Depending on the provider, you will be asked for more details.
 
-Deleting your account in DVC Studio does not delete your Git repositories.
+![](https://static.iterative.ai/img/studio/s3_remote_settings_v2.png)
 
-</admon>
+The credentials must have the required permissions. For
+[data remotes](/doc/studio/user-guide/experiments/configure-a-project#data-remotes--cloud-storage-credentials),
+you can find more details about required permissions [here][data remote].
+
+[data remote]: /doc/user-guide/data-management/remote-storage
+[settings]: https://studio.iterative.ai/user/_/settings
+
+Finally, click `Save credentials`.
+
+## Manage teams
+
+In this section, you can view and manage the teams that you own. You can also
```

Comment:
Question: Does this list also include teams where I am not an admin (owner)?

Reply:
Yes, with a lablel. Updating it to 
`In this section, you can view all the teams you are member of.`

<img width="880" alt="Screenshot 2023-12-21 at 12 36 11" src="https://github.com/iterative/dvc.org/assets/20840228/ae61c68b-57c0-40bd-91d5-5d915dd336e0">